### PR TITLE
Content update - community voting starts; CFP closes

### PR DESCRIPTION
**Pull request by hypha**

### CODE REVIEW

File: data/pages-content/voting.md [hunk 1]
```diff
@@ -4,23 +4,23 @@ title: Voting
...
+Community Voting is open between
 
-Monday, 27 March 14:00:00 CEST and
+Monday, 28 March 10:00:00 CEST and
 
-Sunday, 9 April 23:59:59 [Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
+Sunday, 02 April 23:59:59 [Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
 
-**Voting details will be published soon!**
+<ButtonLink href="https://voting.europython.eu">Take me to vote now!</ButtonLink>
```
Comment (Vaibhavs10) [inline data/pages-content/voting.md:19]: You're probably on it but this currently redirects to EP2022. So we need to ensure that this points to EP2023 when merged.

File: data/pages-content/faq.md [hunk 2]
```diff
@@ -47,20 +49,20 @@ A. The conference will feature around 20 hands-on tutorials & workshops, 120 tal
...
-To select talks, we run a Call for Proposals (CFP). Talks are then curated based on the input from the community voting.
+To select talks, we ran a Call for Proposals (CFP). Talks are then curated based on the input from the community voting.
 
 We are always keen to hear new ideas on how to enrich the programme and other activities you are interested in participating during the conference. Share your thoughts with the programme team at [programme@europython.eu](mailto:programme@europython.eu).
 
 
 ## **Q. When do you accept talk proposals?**
 
-A. The Call for Proposals (CFP) will run for two weeks, plus one week extension between **6-26 March 2023**. We are thrilled to receive proposals on any topic that could be of interest to our community.
+A. The Call for Proposals (CFP) was run between **6-26 March 2023**. We are thrilled to have received more than 500 proposals on a wide range of topics. This record-breaking number of proposals reflected the great width of interests of our community. We are also proud to have provided support to first-time speakers via our [Speaker Mentorship Programme
```
Comment (Vaibhavs10) [inline data/pages-content/faq.md:59]: I think the URL is not complete in the Speaker mentorship program.

**Review by Vaibhavs10 — APPROVED** [review]
Thanks for adding all the info!

File: data/pages-content/voting.md (hunk 1 above)
Comment (artcz) [reply data/pages-content/voting.md:19]: This is correct,  the old app will be gone by then.

File: data/pages-content/faq.md (hunk 2 above)
Comment (hypha) [reply data/pages-content/faq.md]: What an idiot! Fixed with thanks!

**Review by artcz — APPROVED** [review]

**Pull request merged**